src/components/Shared/StatusBadge.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import styled from "react-emotion";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { colors } from "../../constants/colors";
+
+export const StatusBadge = ({
+  icon = null,
+  text = "",
+  backgroundColor = colors.white,
+  textColor = colors.black,
+  size = "sm",
+}) => {
+  const fontConfig = {
+    sm: `1.3rem`,
+    lg: `1.625rem`,
+  };
+
+  const StyledBadge = styled("div")`
```

Comment:
I think this way of styling with react-emotion is cool, but I also think that we might get further faster and maintain more consistency if we try to use Bootstrap font-size utility classes as much as possible. They provide responsive sizing based on existing media queries 

https://getbootstrap.com/docs/4.0/content/typography/

Instead of having the `sm` & `lg` as size props be fixed values, maybe we could have them refer to Bootstrap's typography classes? `lg` could be `h3` and  `sm` could be `h5`.

Just an option, that might not be worth it right now.

Reply:
Good call on the font sizes - I like that better. I updated the styling of the Item title as well and removed the entire `src/styles/WorkOrderInventoryStatus.js` file since it was just setting a single font size. ✂️ 

For the column order, I think I misunderstood the first column part of this requirement.
>  Move status field to the far right (first column)

@JaceDeloney - can you confirm Mateo's comment about the order of these fields above? I think he is right, and I'll fix before merging.

Reply:
Chatted with Jace on Slack, and we sorted this out - merging!